polyfills/Element/prototype/getAttributeNames/polyfill.js

```diff
@@ -0,0 +1,5 @@
+Element.prototype.getAttributeNames = function() {
+    return Array.from(this.attributes).map(function(a) {
```

Comment:
Array.from is a big polyfill, we can avoid the dependency on it by relying only on `Array.prototype.map` like so:
```js
return Array.prototype.map.call(this.attributes, function(a) {
    return a.name;
});
```